tmt/steps/__init__.py

```diff
@@ -75,6 +75,22 @@ def enabled(self):
         except AttributeError:
             return None
 
+    @property
+    def is_in_standalone_mode(self):
```

Comment:
+1 to a common parent from me so that we don't have to duplicate such stuff. I will try to come up with something :)

Reply:
I added the `Phase` class as a common ancestor for `Action` and `Plugin` and also updated the `Step.plugins()` method to `Step.phases()` since it returns a mix of Actions and Plugins, this hopefully removes some possible confusions.